bpmn-model/src/test/java/io/zeebe/model/bpmn/validation/ZeebeValidationTest.java

```diff
@@ -89,11 +89,11 @@
             .startEvent()
             .serviceTask("task", b -> b.zeebeJobType("type"))
             .boundaryEvent("msg1")
-            .message(m -> m.name("message").zeebeCorrelationKey("$.id"))
+            .message(m -> m.name("message").zeebeCorrelationKeyExpression("$.id"))
```

Comment:
This looks like a path expression to me. If we FEELify it, it will become `=$.id` which seems nonsensical; same for `$.orderID` which comes next

Reply:
:+1: I cleaned the test case.